manifests/0000_20_kube-apiserver-operator_08_flowschema.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
+kind: PriorityLevelConfiguration
+metadata:
+  name: openshift-system
+spec:
+  limited:
+    assuredConcurrencyShares: 10
+    limitResponse:
+      queuing:
+        handSize: 6
+        queueLengthLimit: 50
+        queues: 16
+      type: Queue
+  type: Limited
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
+kind: PriorityLevelConfiguration
+metadata:
+  name: openshift-control-plane-operators
+spec:
+  limited:
+    assuredConcurrencyShares: 20
+    limitResponse:
+      queuing:
+        handSize: 6
+        queueLengthLimit: 50
+        queues: 128
+      type: Queue
+  type: Limited
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
```

Comment:
I'd rather keep rules like this local to operators that use them.

Reply:
@deads2k Yes, I thought about it. the only concern I have is the `matchingPrecedence` field. This defines the matching order, so we need to keep in mind all the other values to make sure the right order is maintained.

Reply:
> @deads2k Yes, I thought about it. the only concern I have is the `matchingPrecedence` field. This defines the matching order, so we need to keep in mind all the other values to make sure the right order is maintained.

We did this with conventions in the CVO.  Our payloads are local.  I'm ok moving these in 4.7 instead of 4.6.